### PR TITLE
Documentation PR to address issue #89, #90 and #99

### PR DESCRIPTION
**Pull request by lucdx**
Fixed images as well as the AboutUs landing page.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -33,12 +33,11 @@ image::UiNew.png[]
...
 == Contributors
 
-The full list of contributors for se-edu can be found https://se-edu.github.io/Team.html[here].
+The full list of contributors for ClassRepo can be found https://cs2113-ay1819s1-f10-1.github.io/main/AboutUs.html[here].
 
 == Acknowledgements
 
-Some parts of this sample application was inspired by the excellent
-http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by Marco Jakob
+This application was adapted from AddressBook-Level 3 by se-edu https://github.com/nusCS2113-AY1819S1/addressbook-level3[here].
```
[the remark this answers is not in the document]
Comment (0WN463) [reply README.adoc]: image
Got a correspondence. It is the seedu link

**Comment by 0WN463** [comment]
Also add a `request closure `label to #89 #90 and #99.
And add a comment that mentions this PR in those issues so that Meg can confirm that they are done

**Comment by lucdx** [comment]
Now references correct Seedu. Issue #89 , #90 and #99 should be fixed now.

**Comment by 0WN463** [comment]
See's commented 4 hours ago.
Look at current time
8:44 am.
Standard life of a Uni Student

**Pull request merged**